magefile.go

```diff
@@ -394,6 +396,44 @@ func Package() {
 	packageAgent(platforms, devtools.UseElasticAgentPackaging)
 }
 
+// DownloadManifest downloads the provided manifest file into the predefined folder
+func DownloadManifest() {
```

Comment:
Let's just return an error instead of panicking for every error happening
```suggestion
func DownloadManifest() error {
```